.github/workflows/master.yml

```diff
@@ -8,8 +8,10 @@ jobs:
   buildAndPublish:
     runs-on: ubuntu-latest
     env:
-      BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
-      BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+      ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
+      ARTIFACTORY_API_KEY: ${{ secrets.ARTIFACTORY_API_KEY }}
+      SIGNING_KEY: ${{ secrets.SIGNING_KEY }}
+      SIGNING_PASSWORD: ${{ secrets.SIGNING_PASSWORD }}
       MAVEN_CENTRAL_USER: ${{ secrets.MAVEN_CENTRAL_USER }}
```

Comment:
MAVEN_CENTRAL_XXX should be removed

Reply:
also have a look at the release workfow: it is modified version of that

Reply:
removed